x-pack/legacy/plugins/siem/public/pages/case/components/tag_list/index.tsx

```diff
@@ -54,6 +56,22 @@ export const TagList = React.memo(
         setIsEditTags(false);
       }
     }, [form, onSubmit]);
+    const { tags: tagOptions } = useGetTags();
+    const [options, setOptions] = useState(
+      tagOptions.map(label => ({
+        label,
+      }))
+    );
+
+    useEffect(
+      () =>
+        setOptions(
+          tagOptions.map(label => ({
+            label,
+          }))
+        ),
+      [tagOptions.length]
```

Comment:
@XavierM while just `[tagOptions]` worked in the browser, when I ran tests I got into an infinite loop because it is always considering tagOptions as a new value. Adding `.length` fixes this. Let me know if you have a different way to handle this